libminifi/test/resources/cn.ckey.pem

```diff
@@ -0,0 +1,31 @@
+Bag Attributes
```

Comment:
yes. i just used the toolkit to generate them. Should I provide that info somewhere in the readme?
Owner: CN=test, OU=NIFI
Issuer: CN=localhost, OU=NIFI
Serial number: 15a6753696100000000
Valid from: Wed Feb 22 14:36:44 EST 2017 until: Sat Feb 22 14:36:44 EST 2020